opensuse_buildservice/debian/postinst

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+# postinst script for notqmail
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+qmaildir=/var/qmail
+logdir=/var/log/svc
+cp=/bin/cp
```

Comment:
Are these really needed or can't we use just "cp" and expect it to be in the path?

Reply:
I guess this can be removed. I have this habit of using the system provided cp in shell scripts, to avoid a trojan cp getting executed. I had admins in my company using $HOME/bin, ., etc in the PATH first.

Reply:
since rpm is run as root this shouldn't be an issue.